packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2679,14 +2679,32 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * Admit the guest(s) to the call once they are waiting
+   * Admit the guest(s) to the call once they are waiting.
+   * If the host/cohost is in a breakout session, the locus url
+   * of the session must be provided as the authorizingLocusUrl.
+   * Regardless of host/cohost location, the locus Id (lid) in
+   * the path should be the locus Id of the main, which means the
+   * locus url of the api call must be from the main session.
+   * If these loucs urls are not provided, the function will do the check.
    * @param {Array} memberIds
+   * @param {Object} sessionLocusUrls: {authorizingLocusUrl, mainLocusUrl}
    * @returns {Promise} see #members.admitMembers
    * @public
    * @memberof Meeting
    */
-  public admit(memberIds: Array<any>) {
-    return this.members.admitMembers(memberIds);
+  public admit(
+    memberIds: Array<any>,
+    sessionLocusUrls?: {authorizingLocusUrl: string; mainLocusUrl: string}
```

Comment:
issue: I am worried about the additional complexity here on the admit method. As a user of the SDK I shouldn't need to worry about the details i.e. what the authorizingUrl is. Is it possible to work these parameters out so the user can still call admit(memberIds)?

Reply:
Yes, I agree. Actually, our users can ignore this argument, and the SDK should handle the details. As you can see, sessionLocusUrls is optional, so it can be omitted. And if this argument is not passed in, we have logics to cover the case. So, yes, it is fine to work these parameters out so the user can still call admit. But in case of some special requirements or situations, we allow users to pass sessionLocusUrls argument to fit their businesses. 
![10_27_27](https://user-images.githubusercontent.com/13958569/224878289-8d121f1d-0179-41cd-8ef6-f8877f348e35.jpg)

Reply:
On the sample page, the admitting action  just calls  meeting.admit([participantID]), and it works well
![image](https://user-images.githubusercontent.com/13958569/224879260-644e49da-9ac9-4d27-ad61-efe0fa6ff918.png)